docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,739 @@
+
+Bonus point, we run [React v18](https://github.com/reactwg/react-18) (unreleased) in a nightly build, in strict mode, without any test failures.
+
+> ⚠️ `@mui/styles` is not compatible with strict mode nor concurrent features.
+
+## A new product line: X
+
+<img loading="lazy" src="/static/blog/mui-core-v5/mui-x-blog-hero.png" alt="Mockup of some MUI X components" style="width: 700px; margin-bottom: 16px;" />
+
+We are very excited to introduce a new product line to the MUI family: [**MUI X**](/x/)!
+We have recently released our [first stable version](https://github.com/mui-org/material-ui-x/releases/tag/v4.0.0).
+
+MUI X embodies our initiative to solve the main pain point developers have reported two years in a row during our developer's survey: [2020](/blog/2020-developer-survey-results/#5-how-can-we-improve-material-ui-for-you), [2019](/blog/2019-developer-survey-results/#4-how-can-we-improve-material-ui-for-you).
+We have heard you, you want **more components**.
+
+In practice, there are so many different components that you might need that, our team can't cover it all.
+Instead, we are focusing on the **hardest** and **most requested** components, leaving the long tail to our [community of contributors](/discover-more/related-projects/#components).
+
+We started working on this effort over a year ago, in a new GitHub repository: [mui-org/material-ui-x](https://github.com/mui-org/material-ui-x).
+You might wonder, why work on it as a different product? We quickly realized that the team working on MUI X would need to become, at least, two times larger than the team working on MUI Core.
+Building a great data grid or charting solution takes a considerable amount of time.
+And yet, focusing on MUI Core is **foundamental** for the adoption of MUI.
+
+You can see this separation of purpose more clearly from the vision and mission statmemnts of the two products:
+
+- **MUI Core**:
+  - Vision: Create robust foundations to build UI components. "robust" is measured by how many developers decided to depend on it.
+  - Mission: Unlock design skills to the developers while allowing them to build UIs fast.
+- **MUI X**:
+  - Vision: Create the last UI component library developers will ever need. High quality, consistent, feature-rich, and covering the most frequent/challenging use cases.
+  - Mission: Make building rich, data intensive dynamic apps a breeze.
+
+Taking on the mission MUI X also required to make a major change to our monetization strategy.
+MUI Core has a positive cashflow thanks to its paid templates, design assets, ads on the documentation, and backers/sponsors (❤️). The [MIT](https://tldrlegal.com/license/mit-license) license model works great for it.
+
+However, none of these revenue sources would scale with the outcome and amount of work required for MUI X.
+It's why we are using an open core license model for this new product.
```

Comment:
```suggestion
It's why we are using an [open core license model](https://medium.com/open-consensus/2-open-core-definition-examples-tradeoffs-e4d0c044da7c) for this new product.
```

I personally was not aware of what open-core meant 😅 Here's a cool post I found while researching it, it might be useful for some people.